dhcpv4/dhcpv4_test.go

```diff
@@ -60,44 +45,37 @@ func TestFromBytes(t *testing.T) {
 	data = append(data, []byte{99, 130, 83, 99}...)
 
 	d, err := FromBytes(data)
-	if err != nil {
-		t.Fatal(err)
-	}
-	AssertEqual(t, d.Opcode(), OpcodeBootRequest, "opcode")
-	AssertEqual(t, d.HwType(), iana.HwTypeEthernet, "hardware type")
-	AssertEqual(t, d.HwAddrLen(), byte(6), "hardware address length")
-	AssertEqual(t, d.HopCount(), byte(3), "hop count")
-	AssertEqual(t, d.TransactionID(), uint32(0xaabbccdd), "transaction ID")
-	AssertEqual(t, d.NumSeconds(), uint16(3), "number of seconds")
-	AssertEqual(t, d.Flags(), uint16(1), "flags")
-	AssertEqualIPAddr(t, d.ClientIPAddr(), net.IPv4zero, "client IP address")
-	AssertEqualIPAddr(t, d.YourIPAddr(), net.IPv4zero, "your IP address")
-	AssertEqualIPAddr(t, d.ServerIPAddr(), net.IPv4zero, "server IP address")
-	AssertEqualIPAddr(t, d.GatewayIPAddr(), net.IPv4zero, "gateway IP address")
-	hwaddr := d.ClientHwAddr()
-	AssertEqualBytes(t, hwaddr[:], []byte{0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}, "flags")
+	require.NoError(t, err)
```

Comment:
a nice thing in the existing code being able to specify a string to identify which line actually failed, but I guess we can live with it

Reply:
You can actually do that with `assert`/`require`, I just left most of them out (except for a few) because I thought they were kind of redundant given the tests being kinda small.